app/views/collections/show.html.erb

```diff
@@ -0,0 +1,45 @@
+<div class="container" id="collection" data-controller="collection-updates" dat-collection-updates-collection-id="<%= @collection.id %>">
+  <%= render 'nav' %>
+  <section class="details">
+    <header class="title">
+      <span class="header-text"><%= @collection.name %></span>
+      <span class="deposit-button">
+        <% if allowed_to?(:deposit?, @collection) %>
+          <%= button_tag '+ Deposit to this collection',
+                data: {
+                  destination: new_collection_work_path(@collection),
+                  toggle: 'modal',
+                  target: '#workTypeModal',
+                  action: "work-type#set_collection"
+                },
+                class: "btn btn-primary" %>
+        <% end %>
+      </span
+    </header>
+
+    <div class="tab-content" id="collectionTabs">
+      <div id="details" class="tab-pane fade show active">
+        <%= render Collections::InformationComponent.new(collection: @collection) %>
+        <%= render Collections::DetailComponent.new(collection: @collection) %>
+        <%= render Collections::ReleaseComponent.new(collection: @collection) %>
+        <%= render Collections::TermsOfUseComponent.new(collection: @collection) %>
+        <%= render Collections::ParticipantsComponent.new(collection: @collection) %>
+        <%= render Collections::WorkflowReviewComponent.new(collection: @collection) %>
+      </div>
+
+      <div id="deposits" class="tab-pane fade">
+        <%= render Dashboard::CollectionComponent.new(collection: @collection) %>
+      </div>
+    </div>
+    <section data-controller="work-type">
```

Comment:
why do we need this section.  This seems like it would be confusing to a screen-reader.

Reply:
Without this, the work type modal displays without the type selection area:

without:

<img width="897" alt="Screen Shot 2020-12-03 at 7 15 31 AM" src="https://user-images.githubusercontent.com/2294288/101048352-67731200-3537-11eb-8cd3-afd6bf569abf.png">

With:

<img width="905" alt="Screen Shot 2020-12-03 at 7 15 13 AM" src="https://user-images.githubusercontent.com/2294288/101048378-6b9f2f80-3537-11eb-92d7-a7f170cc4c9c.png">

Reply:
I'm not saying to remove the ` data-controller="work-type"`, but that it shouldn't be a section.  Does it work if you move `data-controller` to the `workTypeModal` div or up to the parent `<section>`